models/Position.js

```diff
@@ -35,7 +35,7 @@ const Position = sequelize.define('position', {
     start_term: {
         type: Sequelize.DATEONLY,
         allowNull: false,
-        defaultValue: Sequelize.NOW,
```

Comment:
shouldn't it be `Sequelize.fn('NOW')` as well?

Reply:
That is no longer a thing, it is now an empty string (like force_ends)

Reply:
it won't pass the validation then right?

Reply:
What validation, there is none yet. Rik made a separate issue for that to make JC happy now